src/components/Dashboard/Dashboard.jsx

```diff
@@ -39,6 +55,10 @@ export const Dashboard = props => {
     }
   }, [props.match]);
 
+  useEffect(() => {
+    document.body.className = theme;
```

Comment:
I like the idea behind this implementation but since this change adds the class at the parent level to all child components, all child elements inherit the styles sourced from the class. 

![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/32549172/e7624e32-463c-41c9-9d35-afb2415de7e1)

Please refer screenshot for the css styles being enforced for Tangible text besides the check box. 

![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/32549172/e2908868-4916-46d0-8e38-b48981f121c0)

You may refer TimeEntryForm.jsx for the html code.

Reply:
@NidaZaki I'm not sure if you've watched my dark mode test video around 1:53 because I implemented this change in another way already.

Reply:
Change looks good. Please also refer the screenshot for the same issue on the text "All the fields are required" and any areas applicable.

![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/32549172/0c47adfc-36fa-4498-94b4-b3b43607d40e)

Reply:
> I like the idea behind this implementation but since this change adds the class at the parent level to all child components, all child elements inherit the styles sourced from the class.
> 
> ![image](https://user-images.githubusercontent.com/32549172/239701806-e7624e32-463c-41c9-9d35-afb2415de7e1.png)
> 
> Please refer screenshot for the css styles being enforced for Tangible text besides the check box.
> 
> ![image](https://user-images.githubusercontent.com/32549172/239701920-e2908868-4916-46d0-8e38-b48981f121c0.png)
> 
> You may refer TimeEntryForm.jsx for the html code.

@NidaZaki Changes for "All the fields are required" have been implemented.
![Screen Shot 2023-05-20 at 3 15 44 PM](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46613683/d70527e6-22f8-4ee0-ac45-d5b3641c3844)